config/development.json

```diff
@@ -38,7 +38,7 @@
 		"CLIENT_SECRET": "erwin-credentials-mock-client-secret"
 	},
 	"INCOMING_REQUEST_TIMEOUT_API": 60000,
-	"NEST_AVAILABLE_LOG_LEVELS": "log,error,warn,debug,verbose,http",
+	"NEST_LOG_LEVEL": "debug",
```

Comment:
please delete so local dev systems use the overall default by default, and only debug if they consciously decide to do so

Reply:
✔️